src/tools/run-make-support/Cargo.toml

```diff
@@ -17,7 +17,7 @@ object = "0.37"
 regex = "1.11"
 serde_json = "1.0"
 similar = "2.7"
-wasmparser = { version = "0.219", default-features = false, features = ["std"] }
+wasmparser = { version = "0.219", default-features = false, features = ["std", "features", "validate"] }
```

Comment:
The latest wasmparser version doesn't pull in ahash or another version of hashbrown.